docs/pages/guides/schema-validation.mdx

```diff
@@ -64,13 +80,140 @@ of the data you want to store in your rooms.
+When a schema is sent to the API it needs to be constructed as a string literal
+to be valid JSON when sent to the API for parsing. In the dashboard, a sample
+schema could be represented as:
+
+```ts
+type Storage {
+  todos: LiveList<LiveObject<Todo>>
+}
+
+type Todo {
+  text: string
+  checked?: boolean
+}
+```
+
+To convert the schema to a string literal which will be used in the `body` key
+of the request, you can use the following code:
+
+```ts
+JSON.stringify({
+  body: `
+type Storage {
+  todos: LiveList<LiveObject<Todo>>
+}
+
+type Todo {
+  text: string
+  checked?: boolean
+}
+`,
+}).replace(/\\n/g, "\n");
+// returns '{"body":"\ntype Storage {\n  todos: LiveList<LiveObject<Todo>>\n}\n\ntype Todo {\n  text: string\n  checked?: boolean\n}\n"}'
+```
+
+To create the schema:
+
+```ts
+fetch("https://api.liveblocks.io/v2/schemas", {
+  method: "POST",
+  body: JSON.stringify({
+    name: "todo-list",
+    body: "\ntype Storage {\n  todos: LiveList<LiveObject<Todo>>\n}\n\ntype Todo {\n  text: string\n  checked?: boolean\n}\n",
+  }),
+});
+```
+
 <Banner title="Schema syntax">
 
 Learn more about our [schema syntax](/docs/guides/schema-validation/syntax) and
 what kind of validations are currently supported.
 
 </Banner>
 
+### Delete a schema
+
+In the dashboard, you can delete a schema by navigating to the schema in
+question and clicking on the trash icon next to the schema version you want to
+delete. If you have attached a schema to a room, you will need to detach the
+schema from the room before you can delete it. If this schema is frozen, the
+trash icon will be greyed out.
+
+<Figure>
+  <video autoPlay loop muted playsInline>
+    <source
+      src="/images/docs/schema-validation/delete-schema.mp4"
+      type="video/mp4"
+    />
+  </video>
+</Figure>
+
+If your schema is in a detached state, you can also delete it by using the
+[delete schema API](/docs/api-reference/rest-api-endpoints#delete-a-schema).
+
+```ts
+fetch("https://api.liveblocks.io/v2/schemas/{id}", {
+  method: "POST",
+});
+```
+
+### Attach a schema to a room
+
+You can attach a schema to a room via the dashboard by navigating to the room in
+question and clicking on the “Attach schema” button.
+
+<Figure>
+  <video autoPlay loop muted playsInline>
+    <source
+      src="/images/docs/schema-validation/attach-schema.mp4"
+      type="video/mp4"
+    />
+  </video>
+</Figure>
+
+Alternatively, you can use the
+[attach schema API](/docs/api-reference/rest-api-endpoints#post-attach-schema-to-room).
+For example:
+
+```ts
+fetch("https://api.liveblocks.io/v2/rooms/{roomId}/schema", {
+  method: "POST",
+  body: JSON.stringify({
+    schema: "todo-list@1",
+  }),
+});
+```
+
+### Detach a schema from a room
+
+You can use the dashboard to remove a schema from a room by navigating to the
+room in question and clicking on the dropdown menu which displays the currently
+attached schema. From there, you can click on the “Detach schema” button.
+
+{" "}
```

Comment:
```suggestion
```

Reply:
prettier added that.  not sure why.  will try to remove